distribution/lib/Standard/Table/0.0.0-dev/docs/api/In_Memory_Table.md

```diff
@@ -3,6 +3,6 @@
 - type In_Memory_Table
     - new columns:(Standard.Base.Data.Vector.Vector Standard.Base.Any.Any) -> (Standard.Table.Table.Table|Standard.Base.Any.Any)
     - to_delimited self delimiter:(Standard.Table.Delimited.Delimited_Format.Delimited_Format|Standard.Base.Data.Text.Text)= quote_style:Standard.Table.Delimited.Quote_Style.Quote_Style= headers:Standard.Table.Headers.Headers= value_formatter:(Standard.Table.Data_Formatter.Data_Formatter|Standard.Base.Nothing.Nothing)= line_endings:(Standard.Base.Data.Text.Line_Ending_Style.Line_Ending_Style|Standard.Base.System.File_Format.Infer)= -> Standard.Base.Data.Text.Text
-- from_java_table java_table:Standard.Base.Any.Any -> (Standard.Table.Table.Table&Standard.Table.In_Memory_Table.In_Memory_Table)
+- from_java_table java_table:Standard.Base.Any.Any -> Standard.Base.Any.Any
```

Comment:
- instead of removing the check
- we could benefit #13390 
- and use `-> Table & Any` to enforce/guarantee conversion to `Table` at least

Reply:
Agree once Table & Any works can do that. But not on this method it should be on the public API and refiners.

Reply:
It should be an internal API but it is used in Google currently. Will add to my list of things to clean up.